api/src/routes/public/status.ts

```diff
@@ -15,25 +14,7 @@ export const statusRoute: FastifyPluginCallbackTypebox = (
   done
 ) => {
   fastify.get('/status/ping', async (req, _reply) => {
-    const url = req.url || 'URL not found';
-    const reqId = req.id || 'REQ_ID not found';
-    const headers = isEmpty(req.headers) ? 'HEADERS not found' : req.headers;
-    const ip =
-      req.headers['x-forwarded-for'] ||
-      req.headers['x-real-ip'] ||
-      req.ip ||
-      'IP not found';
-    const query = isEmpty(req.query) ? 'QUERY not found' : req.query;
-
-    fastify.log
-      .child({
-        URL: url,
-        REQ_ID: reqId,
-        HEADERS: headers,
-        IP: ip,
-        QUERY: query
-      })
-      .debug('returning a ping');
+    req.log.child({ req }).debug('returning a ping');
```

Comment:
```suggestion
    fastify.log.child({ req }).debug('returning a ping');
```

This is not better, but is a clearer mental model. Use `fastify.log` and `fastify.log.child` everywhere.